### ISSUE TITLE
Better mission descriptions

### ISSUE THREAD

**Issue by rnyoakum** [raceintospace/raceintospace#492]
There need to be better mission descriptions in the manual. The **Mission Description** section contains vague notes about some of the mission requirements, but very little about what they actually entail.

Some of my concerns:
* There's a brief mention of the (Orbit) descriptor in the Docking Module section, but it should be (re-)referenced in the Mission Description section. It's described only once, briefly, and is confusing to novice players.
* What is an Orbiting Lab? What steps are involved?
* There are two missions named Joint Manned Orbitals Docking EVA Duration (30 & 35).
* What does it mean that Joint Manned Lunar Orbital: LOR (47) is a joint mission? **Why** is it joint? What's the difference between it and Manned Lunar Orbital (46)? (_Note: In-game it is referred to as an EOR_).

**Comment by peyre** [comment]
Better descriptions in the manual is a good idea.  I'll have to take a look at it.

**Comment by peyre** [comment]
Hm, space is a little tight there, but we could fit it in.  It may not look very good (and would take up a bit of space) but would be much more informative.  I could write up descriptions for the missions and add them.

We could also, maybe, make that window that shows pad letter and mission type, duration, and name into a button that would bring up a help window displaying the same information.

![Future Missions](https://user-images.githubusercontent.com/3118014/97336959-df656280-183c-11eb-9076-edfe507c715e.png)

**Comment by peyre** [comment]
I'll have to fly missions 30 and 35 this weekend to see what exactly they are in reality (vs. what the names claim they are).  Also, wasn't there a mission you pointed out recently that was LOR but claimed to be EOR, or something like that?

    On Sunday, October 25, 2020, 8:58:40 PM PDT, Ryan Yoakum <notifications@github.com> wrote:  
 
 


There need to be better mission descriptions in the manual. The Mission Description section contains vague notes about some of the mission requirements, but very little about what they actually entail.

Some of my concerns:
   
   - There's a brief mention of the (Orbit) descriptor in the Docking Module section, but it should be (re-)referenced in the Mission Description section. It's described only once, briefly, and is confusing to novice players.
   - What is an Orbiting Lab? What steps are involved?
   - There are two missions named Joint Manned Orbitals Docking EVA Duration (30 & 35).
   - What does it mean that Joint Manned Lunar Orbital: LOR (47) is a joint mission? Why is it joint? What's the difference between it and Manned Lunar Orbital (46)? (Note: In-game it is referred to as an EOR).

**Comment by rnyoakum** [comment]
> What does it mean that Joint Manned Lunar Orbital: LOR (47) is a joint mission? Why is it joint? What's the difference between it and Manned Lunar Orbital (46)? (Note: In-game it is referred to as an EOR).

The game data files refer to mission 47 as Joint Manned Lunar Orbital EOR, as does the original BARIS manual, but the [RIS documentation](https://github.com/raceintospace/raceintospace/blob/master/doc/manual/manual.md) refers to it as LOR. I believe this is a typo in the RIS documentation.

**Comment by peyre** [comment]
Ah yeah, that's the one!  I'll have to run it as well and verify what it does in practice.

**Comment by peyre** [comment]
Ok, I've taken a closer look at these three missions.  The Player's Aid Card says 30 is EVA Docking and the 35 is a Docking EVA mission, but in the game they both show as Docking EVA, so they have identical names.  They also fly the same.  It also looks like they're **both** duplicates of 33 Joint Manned Orbitals Docking EVA.  Seems to me we could repurpose both missions.  Maybe one of them could be changed to a Joint Orbital Rendezvous (two craft fly close together but don't dock) that could be done with Mercury/Vostok.

Mission 47 actually shows as EOR, both in the game and the Player's Aid Card, and it does rendezvous in Earth orbit.  What it is is a Manned Docking Test followed by a Lunar Orbital.  Probably we should rename it to Jt Manned Docking Test and Lunar Orbital.  "EOR" doesn't apply, since it's not rendezvouing with another spacecraft--just performing a docking test.

**Comment by peyre** [comment]
I've added descriptions to the manual for all the missions.  Doubtless they can be fine-tuned and you may want to edit a number of them yourself, but it's a start.

**Comment by peyre** [comment]
Hey, there's also mission 44: Joint Manned Lunar Pass: EOR.  This one's kind of pointless.  If you schedule it for an Apollo, part B launches the spacecraft and part A launches a DM, which you don't need.  If you schedule it for a Gemini, part A still launches a DM, and part B launches both the Gemini and the kicker.  I'd say we can totally repurpose this number for another mission.  Possibly an artificial gravity experiment like we performed on Gemini 11, which could earn you prestige.

**Comment by peyre** [comment]
Ok, the name box in Future Missions is now a button that brings up a description of the currently selected mission.

![Description](https://user-images.githubusercontent.com/3118014/98869065-04a1c580-2426-11eb-84f9-890800d32f18.png)

**Comment by peyre** [comment]
I've written up the oddball missions as a separate Issue, so I'll close this one since the descriptions are now implemented in the manual and Future Missions.  They can always be fine-tuned, of course, but the main task is done.
